app/dashboard/static/js/app/tables/common.js

```diff
@@ -278,9 +278,6 @@ define([
             if (type === 'display') {
                 node = _dateNode(date).firstElementChild;
                 rendered = node.outerHTML;
-            } if (type === 'sort') {
```

Comment:
Also it looks like there was a missing `else` before the `if` - let's see what that does on staging.

I would guess that if type is 'display', the `if (type === 'sort')` would evaluate to false and the `else` branch would replace `rendered`.  With this change, only the actual block under `if (type === 'display')` will be run and `rendered` will be the HTML rather than an ISO date.

Reply:
@gctucker I am not sure if we are missing something here, I've checked past code..and found which change caused the bug, but after you check back-end logs let me know if anything else will need to change.